tests/src/test/scala/actionContainers/PythonActionContainerTests.scala

```diff
@@ -162,9 +161,14 @@ class PythonActionContainerTests extends BasicActionRunnerTests with WskActorSys
     }
 
     it should "run zipped Python action containing a virtual environment" in {
-        val zippedPythonActionWrongName = TestUtils.getTestActionFilename("action_python_virtualenv.zip")
-
-        val code = readAsBase64(Paths.get(zippedPythonActionWrongName))
+        var zippedPythonActionName = ""
+        if (imageName == "python2action") {
+            zippedPythonActionName = TestUtils.getTestActionFilename("action_python2_virtualenv.zip")
+        }
+        if (imageName == "python3action") {
+            zippedPythonActionName = TestUtils.getTestActionFilename("action_python3_virtualenv.zip")
+        }
```

Comment:
You can use a scala idiomatic way to no use a variable (mutability is to be carefully chosen and actually not needed here).

In scala `if` is an expression, which means it does return a value from each branch instead of just executing it. With that in mind, you can rewrite your code to:

```
val zippedPythonActionName = if (imageName == "python2action") "action_python2_virtualenv.zip" else "action_python3_virtualenv.zip"
val zippedPythonAction = TestUtils.getTestActionFilename(zippedPythonActionName)
```

Does that make sense?